tests/general/AnnotationBearTest.py

```diff
@@ -112,3 +114,14 @@ def test_combined_strings(self):
             self.assertIn(string2, results[0].contents['strings'])
             self.assertIn(string3, results[0].contents['strings'])
             self.assertEqual(results[0].contents['comments'], ())
+
+    def test_external_coalang(self):
+        section3 = Section("")
```

Comment:
but wouldn't that change it for further tests? if i use the section in the setup?